SplunkAppForWazuh/appserver/static/js/controllers/settings/index/settingsIndexCtrl.js

```diff
@@ -20,41 +20,83 @@ define([
       this.currentDataService = $currentDataService
       this.urlTokenModel = $urlTokenModel
       this.notification = $notificationService
+      this.selectedIndex = 'wazuh'
 
       this.dropdown = new Dropdown(
         'inputIndexes',
-        `| metasearch index=* sourcetype=wazuh | stats count by index, sourcetype | fields index`,
+        `| metasearch index=* sourcetype=*wazuh* | stats count by index, sourcetype | fields index`,
```

Comment:
**nitpick**: The reference to `wazuh` should be variable?

Reply:
I think it's not necessary for this, it's just the minimum requirement to consider an index as valid.